### PR TITLE
Fix profile link

### PR DESCRIPTION
**Pull request by susiecoleman**
Use a different link to access profile depending on whether you're on CODE or PROD

Open public profile in new tab

### CODE REVIEW

File: conf/DEV.conf [hunk 1]
```diff
@@ -13,5 +13,8 @@ identity-admin {
     baseUrl = ${identity-admin.adminApi.baseRootUrl}"/v1"
     secret = "secret"
   }
+  editUser {
+    baseProfileUrl = "http://profile.theguardian.com/user/id/"
```
Comment (jamespamplin) [inline conf/DEV.conf]: https?

**Comment by jamespamplin** [comment]
:+1: 1 small change required, and conflicts, otherwise looks good to me

File: conf/DEV.conf (hunk 1 above)
Comment (susiecoleman) [inline conf/DEV.conf]: fixed

**Comment by jamespamplin** [comment]
:+1:

**Pull request merged**